staging/src/k8s.io/kubectl/pkg/util/i18n/translations/kubectl/zh_CN/LC_MESSAGES/k8s.po

```diff
@@ -100,16 +100,16 @@ msgid ""
 "\t\tkubectl top pod -l name=myLabel"
 msgstr ""
 "\n"
-"\t\t# 显示 default namespace 下所有 pods 下的 metrics\n"
+"# 显示默认命名空间中所有 Pod 的指标\n"
```

Comment:
I don't think “默认” should be translated into "default", otherwise users need to think about what the default namespace is.

I don't think “namespace” should be translated, and even if it is translated, please follow the [Chinese localization terminology guide ](https://kubernetes.io/zh/docs/contribute/localization_zh/#%E6%9C%AF%E8%AF%AD%E5%AF%B9%E7%85%A7)

/cc @tengqm